code/game/turfs/simulated/floor/plating/asteroid.dm

```diff
@@ -101,9 +101,19 @@
 	return
 
 /turf/open/floor/plating/asteroid/singularity_act()
+	src.ChangeTurf(/turf/open/space)
```

Comment:
@JJRcop 

Singulo already eats to baseturf, the asteroids baseturf is itself though